pkg/controller/registry/reconciler/configmap.go

```diff
@@ -26,26 +26,36 @@ type configMapCatalogSourceDecorator struct {
 	*v1alpha1.CatalogSource
 }
 
+const (
+	// ConfigMapServerPostfix is a postfix appended to the names of resources generated for a ConfigMap server.
+	ConfigMapServerPostfix string = "-configmap-server"
+)
+
 func (s *configMapCatalogSourceDecorator) serviceAccountName() string {
-	return s.GetName() + "-configmap-server"
+	return s.GetName() + ConfigMapServerPostfix
 }
 
 func (s *configMapCatalogSourceDecorator) roleName() string {
-	return s.GetName() + "-configmap-reader"
+	return s.GetName() + ConfigMapServerPostfix
```

Comment:
This is wrong. `ConfigMapServerPostfix` ("-configmap-server") should not be used here.

Add a new constant, `ConfigMapReaderPostfix`, set to "-configmap-reader".